dialogue/TownNPCs/Mocha/RandonBananaAzuraPistachio/BananaCardinalPistachio_Mocha_Random2.json

```diff
@@ -28,12 +28,22 @@
       "Así que pásate por aquí cuando quieras, @CharacterName.",
       "Podría hasta darte algún descuento si decides alquilar un cuarto..."
     ],
+    "pl": [
+      "Zanim odziedziczyłam tą karczmę od mojego ojca, <c=hickory>Hickory</c>...",
+      "Miałam problemy rozmawiając z klientami, kiedy mu pomagałam w karczmie.",
+      "Było mi trudno nauczyć się, jak mówić do nowymi klientami grzecznie i bez aksentem...",
+      "ale po jakimś czasie, to już była łatwa sprawa!",
+      "Jest relaksujące, że nie muszę martwić się o tym, gdy jestem z wami.",
```

Comment:
"To relaksujące, że nie musze się o to martwić, gdy jestem z wami." will sound more natural